snorkel/labeling/model/label_model.py

```diff
@@ -147,18 +210,21 @@ def _build_mask(self) -> None:
                     self.mask[sj:ej, si:ei] = 0
 
     def _generate_O(self, L: Matrix, higher_order: bool = False) -> None:
-        """Form the overlaps matrix, which is just all the different observed
-        combinations of values of pairs of sources
-
-        Note that we only include the k non-abstain values of each source,
-        otherwise the model not minimal --> leads to singular matrix
+        """Generate overlaps and conflicts matrix from label matrix.
+
+        Parameters
+        ----------
+        L
+            An [n,m] label matrix with values in {0,1,...,k}
+        higher_order: optional
+            Whether to include higher-order correlations (e.g. LF pairs) in matrix, by default False
         """
         L_aug = self._get_augmented_label_matrix(L, higher_order=higher_order)
         self.d = L_aug.shape[1]
         self.O = torch.from_numpy(L_aug.T @ L_aug / self.n).float()
 
     def _init_params(self) -> None:
-        """Initialize the learned params
+        r"""Initialize the learned params.
```

Comment:
regex?

Reply:
pydocstyle complains if there's `\` in the docstring, should I wrap math in something to prevent the error?

Reply:
which error code is it?